tests/integration/.env.example

```diff
@@ -0,0 +1,8 @@
+# Make sure that regserver is suffixed with /connect 
+# E.g. https://scc.suse.com/connect
+SCC_REGSERVER_URL=
+SCC_USERNAME=
+SCC_PASSWORD=
+SMT_DB_USER=
```

Comment:
Arbitrary credentials -- the DB container appears to use anything provided in the .env file.